src/Model/Table/PermissionsTable.php

```diff
@@ -177,7 +177,11 @@ public function allow($aro, $aco, $actions = '*', $value = 1)
             $save = array_combine($permKeys, array_pad([], count($permKeys), $value));
         } else {
             if (!is_array($actions)) {
-                $actions = ['_' . $actions];
+                if ($actions{0} !== '_') {
```

Comment:
I think it would be better to either use square brackets, or `strpos('_', $actions) !== 0`. Using braces for string accessing is a very unknown php feature (I actually had to go and search for it)